src/views/VallisFish.vue

```diff
@@ -2,171 +2,141 @@
+              <td rowspan="3">
+                <b-btn :id="`${fish.unique.name}_unique_tooltip`" size="md" variant="link" class="m-3">
+                  {{ fish.unique.name }}
+                </b-btn>
+                <b-tooltip :target="`${fish.unique.name}_unique_tooltip`" placement="top">
+                  <FishImg
+                    type="parts"
+                    :item="fish.unique.thumb"
+                    :name="fish.unique.name"
+                    width="200"
+                    v-if="fish.unique.thumb"
+                  />
+                  <span v-else> No image available</span>
+                  <b-btn
+                    :href="fish.unique.wiki"
+                    target="_blank"
+                    size="sm"
+                    rel="noopener"
+                    variant="link"
+                    v-if="fish.unique.thumb"
+                  >
+                    Wikia Article
+                  </b-btn>
+                </b-tooltip>
+              </td>
+              <td>{{ fish.small.standing }}</td>
+              <td rowspan="3">{{ fish.location }}</td>
+              <td rowspan="3">{{ fish.time }}</td>
+              <td rowspan="3">
+                <span>{{ fish.rarity }}</span>
+
+                <br v-if="fish.hotspot" />
+                <span v-if="fish.hotspot">Hotspot Required</span>
+
+                <br v-if="fish.bait" />
+                <b-btn
+                  v-if="fish.bait"
+                  :id="`${fish.name}_${fish.bait.name}_bait_tooltip`"
+                  size="md"
+                  variant="link"
+                  class="mb-1"
+                >
+                  {{ fish.bait.name }}
+                </b-btn>
+                <b-tooltip v-if="fish.bait" :target="`${fish.name}_${fish.bait.name}_bait_tooltip`" placement="top">
+                  <FishImg
+                    type="bait"
+                    :item="fish.bait.thumb"
+                    :name="fish.bait.name"
+                    width="200"
+                    v-if="fish.bait.thumb"
+                  />
+                  <span v-else> No image available</span>
+                </b-tooltip>
+              </td>
+              <td rowspan="3">
+                <span v-for="spear in fish.spear" :key="spear">{{ spear }} <br /></span>
+              </td>
+              <td rowspan="3">{{ fish.maximumPoint }}</td>
+            </tr>
+            <tr :class="'color' + ((index % 2) + 1)" v-if="fish.medium">
+              <td>Adorned</td>
+              <td>{{ fish.medium.resources.scrap }}</td>
+              <td>{{ fish.medium.standing }}</td>
+            </tr>
+            <tr :class="'color' + ((index % 2) + 1)" v-if="fish.large">
+              <td>Magnificent</td>
+              <td>{{ fish.large.resources.scrap }}</td>
+              <td>{{ fish.large.standing }}</td>
+            </tr>
+          </tbody>
+        </table>
       </div>
+
+      <router-link to="/poe/fish/howto#hotspots">
```

Comment:
Linking back to the PoE article, whose hotspots look different, may confuse readers.